charts/ix-chart/2111.0.2/Chart.yaml

```diff
@@ -15,7 +15,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 2111.0.1
+version: 2111.0.2
```

Comment:
Let's please use the version `2112.0.0` - first entry is `2112` with `21` being the year and `12` being the month. For the second one - we bump it when we introduce a new feature/improvement in the same month and the last is for bug fixes.

Reply:
Got it, I've made updates accordingly. Thanks